.github/issue-labler.yml

```diff
@@ -0,0 +1,3 @@
+Status: 
+  Triage:
+    - '(Describe the bug)'
```

Comment:
```suggestion

```
What is the use of this file? I don't think we need this file, as we want to add labels to all issues, not just to specific types.

Reply:
It's configuration.  See https://github.com/github/issue-labeler

Reply:
@gregsdennis, do we need to add this label only for bugs? It will not add a label for the other issues. 
Also, I have tried testing this code in my repo, but it's not working. @Min2who, could you please share the workflow run link? Here is the link to [my run](https://github.com/princerajpoot20/CI-CD-Testing/actions/runs/8032729704/)

Reply:
See the heading "Label All Issues" in the linked docs.

Reply:
@gregsdennis Sorry for this, my bad for not getting this. 
Just one thing to confirm. In the documentation of `all labels`, you have to write `/.*/`. If we write to Describe the bug, it will be the case of a simple example. Sorry if I am missing something.
<img width="856" alt="Screenshot 2024-02-25 at 1 16 59 AM" src="https://github.com/json-schema-org/website/assets/44585452/b9604b63-9a51-44bd-ae27-3fe448108dea">

Reply:
The requirement is to add the label to all issues.  The `/.*/` text is just a regex that matches everything.

The config specifies a label and a regex to match. If the regex matches the content of the issue, the label is applied.

So by specifying a regex that matches everything, the label is applied to all issues, which is the requirement.